lib/matplotlib/tests/test_axes.py

```diff
@@ -8272,3 +8272,27 @@ def test_bar_all_nan(fig_test, fig_ref):
 
     ax_ref.bar([1], [1]).remove()
     ax_ref.bar([1], [1])
+
+
+@image_comparison(["extent_units_y_axis.pdf"], style="mpl20")
```

Comment:
I would change these to png - many tests are not run if the pdf converter is not found on an architecture.   While you are changing it, I would also just merge these two images on one figure as separate subplots.

Reply:
subplots sound good :)
have made the changes.